src/internal/packager/images/pull.go

```diff
@@ -168,3 +169,63 @@ func (i *ImgConfig) PullImage(src string, spinner *message.Spinner) (img v1.Imag
 
 	return img, nil
 }
+
+// IndexJSON represents the index.json file in an OCI layout.
+type IndexJSON struct {
```

Comment:
In `features/oci-publish` I import the OCI spec library, once merged into the main feat branch, could replace this strcut w/ `ocispec.IndexManifest` (can't remember the specific name atm).